code/game/machinery/constructable_frame.dm

```diff
@@ -754,6 +754,15 @@ obj/item/weapon/circuitboard/rdserver
 							"/obj/item/weapon/stock_parts/matter_bin" = 1,
 							"/obj/item/weapon/stock_parts/capacitor" = 2)
 
+/obj/item/weapon/circuitboard/box_cloner
+	name = "Circuit Board (Box Cloner)"
+	build_path = "/obj/machinery/egg_incubator/box_cloner"
+	origin_tech = "illegal=3"
```

Comment:
I would not do this. The current source of illegal tech is already locked behind research and materials because it allows for very strong items - the possibility of someone getting this board and then duplicating it in the genfab for instant cheap illegal levels would break that flow.